pkg/generators/openapi.go

```diff
@@ -353,10 +353,49 @@ func (g openAPITypeWriter) generateCall(t *types.Type) error {
 	return g.Error()
 }
 
+func (g openAPITypeWriter) generateValueValidations(vs *ValueValidations) {
+
+	if vs == nil {
+		return
+	}
+	if vs.Minimum != nil {
+		g.Do("Minimum: util.PointerToFloat64($.Minimum$),\n", vs)
```

Comment:
An alternative to using these helper functions is to do as you did in the test by abusing the array literal notation:

```
&[]float64{20.0}[0]
```

/cc @apelisse WDYT is preferable